tfworker/constants.py

```diff
@@ -16,13 +16,11 @@
 
 _CWD = os.getcwd()
 
-DEFAULT_BACKEND_BUCKET = "tfworker-terraform-states"
+
 DEFAULT_BACKEND_PREFIX = "terraform/state/{deployment}"
 DEFAULT_CONFIG = f"{_CWD}/worker.yaml"
 DEFAULT_REPOSITORY_PATH = _CWD
-DEFAULT_AWS_REGION = "us-west-2"
-DEFAULT_GCP_REGION = "us-west2b"
-DEFAULT_BACKEND_REGION = "us-west-2"
-DEFAULT_TERRFORM = "/usr/local/bin/terraform"
+DEFAULT_AWS_REGION = "us-east-1"
+DEFAULT_GCP_REGION = "us-east-1a"
```

Comment:
Why do we have defaults for these?

Reply:
remnant of early code i think. i can remove.

Reply:
I don't have a strong opinion either way.

Reply:
kk. leaving it for now. i'd like to include that change in a larger PR for handling argument validation outside of click (since we're now allowing args to be defined in the config).